src/mon.c

```diff
@@ -2151,13 +2151,17 @@ register struct monst *mtmp;
         struct obj* list = collect_all_transient(NULL);
         struct obj* next;
         if (list) {
-            if (Blind && !Deaf)
-                You("hear a rush of objects jumble together nearby.");
-            else if (Blind && Deaf)
-                You_feel("a jarring vibration nearby.");
-            else
+            if (!Deaf) {
```

Comment:
While we're here, we may want to add some message (if not Deaf) to the effect of hearing a huge amount of cheering after the NPC is killed (though perhaps not; it might be too many messages.)

Reply:
i'd want those NPCs to also cheer if the player is killed, and i don't feel confident that i'd be able to code that!